src/datagrid/DataGrid.tsx

```diff
@@ -228,6 +228,18 @@ export enum GridRowType {
 }
 
 const isSelectedKey = "isSelected";
+
+// Default label for datagrid items
+const DEFAULT_ITEM_TEXT: string = "items";
```

Comment:
Why do we need defauly text ?  isnt this something which should come as Prop just like other labels ?

Reply:
@maheshkulkarni4 This it default text that will get displayed in footer if user has not send any label to display. 
For example, 10 items vs 10 users. Should I make it a mandatory prop ?